src/components/Notes/Notes.css

```
@@ -23,15 +27,62 @@
 .note-list-container * {
   text-align: left;
   margin: 0;
+  min-width: 260px;
+  max-width: 95vw;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
 }
 
+.note-item-info {
+  /* border: 1px solid black; */
+  padding-left: 10px;
+  padding-bottom: 12px;
+  padding-top: 4px;
+  max-height: 30px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  cursor: pointer;
+}
 .note-item {
   align-self: stretch;
   display: flex;
   justify-content: space-between;
-  border: 1px solid;
-  border-radius: 0.5rem;
-  padding: 0.5rem;
+  /* border-bottom: 1px solid #202833; */
+}
+
+.note-item-open {
+  background: url(../../assets/img/arrow-right.png);
+  background-repeat: no-repeat;
+  background-size: 18px 18px;
+  background-position: right;
+  border: none;
+  height: 30px;
+  min-width: 40px;
+  /* height: 2%; */
+  /* padding: 10px; */
+}
+
+.note-item-title {
+  color: #202833;
+  padding-bottom: 4px;
+  white-space: nowrap;
+  overflow: hidden;
+  /* max-height: 100px; */
+  /* text-overflow: ellipsis; */
+  /* overflow-wrap: anywhere; */
+}
+
+.note-item-content {
+  font-size: 12px;
+  /* border: 1px solid black; */
+  padding-bottom: 10px;
+  max-width: 100%;
+  white-space: nowrap;
+  overflow: hidden;
```

Letter "p" is cut off so maybe add this
```suggestion
  overflow: hidden;
  line-height: 12px;
```